docs/source/models.rst

```diff
@@ -1634,6 +1634,29 @@ been logged: 'baseline_model' and 'candidate_model' for comparison purposes in t
 Additional information about model evaluation behaviors and outputs is available in the
 :py:func:`mlflow.evaluate()` API docs.
 
+.. note:: Differences in the computation of Area under Curve Precision Recall score (metric name
+    ``precision_recall_auc``) between multi and binary classifiers:
+
+    Multiclass classifier models, when evaluated, utilize the standard scoring metric from sklearn:
+    ``sk_metrics.roc_auc_score`` to calculate the area under the precision recall curve. This
```

Comment:
```suggestion
    ``sklearn.metrics.roc_auc_score`` to calculate the area under the precision recall curve. This
```

`sk_metrics` sounds like a third-party packages that provides functions to compute metrics.